app/src/main/java/com/google/samples/apps/sunflower/compose/home/HomeScreen.kt

```diff
@@ -160,47 +164,48 @@ fun HomePagerScreen(
         }
     }
 }
-
-@OptIn(ExperimentalFoundationApi::class)
-@Composable
-private fun HomeTopAppBar(
-    pagerState: PagerState,
-    onFilterClick: () -> Unit,
-    modifier: Modifier = Modifier
-) {
-    TopAppBar(
-        title = {
-            Row(
-                Modifier.fillMaxWidth(),
-                horizontalArrangement = Arrangement.Center,
-            ) {
-                Text(
-                    text = stringResource(id = R.string.app_name)
-                )
-            }
-        },
-        modifier.statusBarsPadding(),
-        actions = {
-            if (pagerState.currentPage == SunflowerPage.PLANT_LIST.ordinal) {
-                IconButton(onClick = onFilterClick) {
-                    Icon(
-                        painter = painterResource(id = R.drawable.ic_filter_list_24dp),
-                        contentDescription = stringResource(
-                            id = R.string.menu_filter_by_grow_zone
-                        ),
-                        tint = MaterialTheme.colors.onPrimary
-                    )
-                }
-            }
-        },
-        elevation = 0.dp
-    )
-}
+// TODO: delete since it wasn't used?
+//
+//@OptIn(ExperimentalFoundationApi::class)
+//@Composable
+//private fun HomeTopAppBar(
```

Comment:
We can use this now since you're migrating to M3 (would ned to update `HomeScreen` to use `Scaffold`)